dotCMS/src/main/java/com/dotcms/integritycheckers/ContentFileAssetIntegrityChecker.java

```diff
@@ -72,7 +72,7 @@ public boolean generateIntegrityResults(final String endpointId) throws Exceptio
     }
 
     @Override
-    public void executeFix(final String endpointId) throws DotDataException, DotSecurityException {
+    public void executeFix(final String remoteIP) throws DotDataException, DotSecurityException {
```

Comment:
In all these cases, I think we should use the JWT fingerprint as the key for the fixes, not the remote IP - or we should use the clusterId of the two environments.  The remoteIp can be changeable and does not really represent a dotCMS installation.